pipelines/rj_smtr/utils.py

```diff
@@ -167,7 +167,7 @@ def map_dict_keys(data: dict, mapping: dict) -> None:
 
 
 def connect_ftp(
-    secret_path: str = constants.FTPS_SECRET_PATH.value, secure: bool = True
+    secret_path: str = constants.RDO_FTPS_SECRET_PATH.value, secure: bool = True
```

Comment:
não deixar padrão pois não é genérico, explicitar na hora de chamar a task
```suggestion
    secret_path: str = None, secure: bool = True
```